hpx/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2021 Nanmiao Wu
+# Copyright (c) 2021 Nikunj Gupta
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
```

Comment:
Would you be ok with using the Task Bench license?

Reply:
no problem.

Reply:
Sorry, actually this is still here (in the `CMakeLists.txt`). I see the `.cc` file changed though.

Reply:
@elliottslaughter Sorry for missing that. I removed the boost license.